### DISCUSSION TITLE
Install

### DISCUSSION THREAD

**Discussion by Tysau1812** [termux/termux-app#3028] [Q&A] [answered]
How do I install this app I'm not understanding .I downloaded universal APK on my one plus 9 it said installed them the widget never appeared in my drawer

**Comment by agnostic-apollo** [comment, marked as answer]
Downloading doesn't automatically install the app. You need to open the apk and install it with package manager after giving permission to calling app to be able to install apks.

https://www.xda-developers.com/how-to-sideload-install-android-app-apk/#apkinstall
